File: CursoPOOUber/JS/Account.js
//Definition of the "Account" class.
class Account {

    constructor(name, document) {
        this.id;
        this.name = name;
        this.document = document;
        this.email;
        this.password;
    }

    printDataDriver() {
        console.log(this.name)
        console.log(this.document)
        console.log(this.email)
        console.log(this.password)
    }
}

File: CursoPOOUber/Java/Account.java
package CursoPOOUber.Java;

//Definition of the "Account" class.
public class Account {
    Integer id;
    public String name;
    public String document;
    public String email;
    public String password;

    public Account(String name, String document) {
    this.name = name;
    this.document = document;
    }

    void printDataDriver(){
        System.out.println("Name: " + name + " Document: " + document + " E-mail: " + email + " Password: " + password);
    }
}
